### DISCUSSION TITLE
How do I disable checking of imports?

### DISCUSSION THREAD

**Discussion by OliverFM** [microsoft/pyright#2612] [Q&A] [answered]
My python code is packaged with Bazel -- is it possible to disable the warning if pyright can't find the relevant import?

**Comment by erictraut** [comment, marked as answer]
I presume that you're referring to an "Import x could not be resolved" diagnostic? That's controlled by the `reportMissingImports` diagnostic rule, which you can disable if you would like.

**Reply by OliverFM** [reply]
Thanks!

**Reply by OliverFM** [reply]
I am trying to set this in my vimrc but it doesn't seem to work:
```let g:ale_python_pyright_config = {'reportMissingImports': 'false'}```
Do you know how I should be setting this up?

**Reply by erictraut** [reply]
Please refer to the configuration documentation [here](https://github.com/microsoft/pyright/blob/main/docs/configuration.md).

**Reply by OliverFM** [reply]
Thanks I added `{ "pythonVersion": "3.8", "reportMissingImports": false}`to the root of my python repo and this solved the issue!
